### ISSUE TITLE
Prop check warn if max is null

### ISSUE THREAD

**Issue by icebob** [monterail/vue-multiselect#88]
I got a vue-warn if `max` property is `null` because in the source code the `max` prop default value is a boolean
https://github.com/monterail/vue-multiselect/blob/master/src/multiselectMixin.js#L171

```
[Vue warn]: Invalid prop: type check failed for prop "max". Expected Number, got Boolean. (found in component: <multiselect>)
```

**Comment by shentao** [comment]
I guess I will change it to default 0, as it should evaluate to false. This is introduced by stricter props checking in Vue in more recent versions.
Thanks for pointing this out!

**Comment by shentao** [comment]
Fixed. Should be available on Wednesday.

**Comment by lionel-bijaoui** [comment]
Thank you !

**Comment by shentao** [comment]
Finally published and available in v1.1.3!
